dsme/dsme-server.c

```diff
@@ -34,13 +34,13 @@
 #define _GNU_SOURCE
 #endif
 
-#include "dsme/mainloop.h"
-#include "dsme/modulebase.h"
-#include "dsme/dsmesock.h"
-#include "dsme/protocol.h"
-#include "dsme/logging.h"
-#include "dsme/messages.h"
-#include "dsme/oom.h"
+#include "../include/dsme/mainloop.h"
+#include "../include/dsme/modulebase.h"
+#include "../include/dsme/dsmesock.h"
+#include <dsme/protocol.h>
```

Comment:
Style: Maybe group these in private / public headers (and put the "public" ones first, so preprocessor stuff in private headers doesn't affect the public headers):

```
#include <dsme/messages.h>
#include <dsme/protocol.h> 

#include "../include/dsme/mainloop.h"
#include "../include/dsme/modulebase.h"
#include "../include/dsme/dsmesock.h"
#include "../include/dsme/logging.h"
#include "../include/dsme/oom.h" 
```